recipes/google-cloud-cpp/all/conanfile.py

```diff
@@ -100,19 +100,18 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
 
     def package_info(self):
-        # Using componen google-cloud-common, so it reduces the risk of conflict (pkgconfig generator)
-        self.cpp_info.components["google-cloud-common"].requires = ["abseil::absl_any", "abseil::absl_flat_hash_map", "abseil::absl_memory", "abseil::absl_optional", "abseil::absl_time"]
-        self.cpp_info.components["google-cloud-common"].libs = ["google_cloud_cpp_common"]
-        self.cpp_info.components["google-cloud-common"].names["cmake_find_package"] = "common"
-        self.cpp_info.components["google-cloud-common"].names["cmake_find_package_multi"] = "common"
+        # FIXME: Components require names["pkg_config"] = "???" https://github.com/googleapis/google-cloud-cpp/search?p=1&q=pc
```

Comment:
Why not fix all pkgconfig names in this PR? It's quite easy, just take a look at installed pkgconfig files.